src/Core/Content/Property/Aggregate/PropertyGroupTranslation/PropertyGroupTranslationEntity.php

```diff
@@ -22,6 +22,11 @@ class PropertyGroupTranslationEntity extends TranslationEntity
      */
     protected $description;
 
+    /**
+     * @var int|null
+     */
+    protected $position;
```

Comment:
position should not be nullable, please add a database and definition default.

Reply:
getDefaults and database default value added.